code/modules/recycling/disposal/bin.dm

```diff
@@ -1,5 +1,6 @@
 // Disposal bin and Delivery chute.
-
+#define CONTAINS_ANIMAL_CHANCE 5
```

Comment:
Metastation contains 53 disposal bins... two and a half stoats per map is _probably_ fine I guess?
Might be better to make this an independent proc somewhere else that selects a specific number (or up to that number) of disposal bins on the station z level after initialisation for finer control though.

Additionally this can also run on any disposal bin that's in a space ruin or something... although that's actually quite funny I think stoats die in space and a lot of these places will be airless.

Reply:
probability will inevitably create enough stoat infestations to where the gimmick runs dry so might be more valuable to introduce more scarcity in a specialized proc, maybe hardcap the spawnable stoats on world init to 1-2 across all z-levels and if people want to flood the station with stoats they can do cargo crates or something